app/models/carbon_source.rb

```diff
@@ -4,7 +4,9 @@ class CarbonSource < ActiveRecord::Base
     "Personal Car", "Company Car"]
 
   validates :source, presence: true
-  validates :conversion_factor, presence: true, numericality: true
+  validates :conversion_factor, presence: true, numericality: {
+    greater_than: 0, message: "must be a positive number"
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.